mlc_config.json

```diff
@@ -29,6 +29,9 @@
         },
         {
             "pattern": "/builtins"
+        },
+        {
+            "pattern": "https://www.10xeditor.com/"
```

Comment:
Ignored because of https://github.com/gaurav-nelson/github-action-markdown-link-check/issues/170